scraper.rb

```diff
@@ -40,6 +82,89 @@ class Governor < Scraped::HTML
   field :party do
     noko.css('td')[4].text.tidy
   end
+
+  # Get extra fields from the Pombola data, if present:
+
+  field :email do
+    pombola_contact_details_of_type('email')
+  end
```

Comment:
When I was having a look at this there was the odd thing in email that wasn't actually a valid email e.g. 'foo'. I wonder if some basic sanity checking here might be in order?